types/sip.js/index.d.ts

```diff
@@ -231,6 +231,16 @@ export namespace WebRTC {
     }
 }
 
+export interface TransportOptions {
+    wsServers?: string | string[];
```

Comment:
Seems like this can also be an `object[]` as shown [here](https://sipjs.com/api/0.11.0/ws_transport_configuration_parameters/#wsservers).